examples/progress.js

```diff
@@ -0,0 +1,49 @@
+/* @jsx h */
+const {h, mount, Component, Text, Bar} = require('../');
+
+const MAX = 20;
+
+class ProgressApp extends Component {
+	constructor() {
+		super();
+
+		this.state = {
+			i: 0
+		};
+	}
+
+	render() {
+		const text = `Running `;
```

Comment:
It'd have to know what else is on the line, or take the other content as a prop and render it itself, and it couldn't be custom components, and it might be a <span> with multiple <Text> to have different colors.

It could have you either pass leftText (string) or leftSize (number), and if you pass leftText it'd render it. This seems confusing though.